### DISCUSSION TITLE
Dag gets stuck in "Scheduled" State when scheduling manually

### DISCUSSION THREAD

**Discussion by saip50** [apache/airflow#19591] [Q&A]
Hi, we are using airflow 2.0.1 to manage our data jobs and automate flow. However, from past a week, DAG's got stuck in scheduled state when we are triggering it manually. As this behavior is observed in all our DAG's. The setup has a default_pool of 128 slots among which only 8 slots have been used. CPU utilisation of the scheduler is also moderate. Also updated,
AIRFLOW__SCHEDULER__MIN_FILE_PROCESS_INTERVAL is set to 30
AIRFLOW__SCHEDULER__DAG_DIR_LIST_INTERVAL is set to 10
And, logs doesn't say anything regarding the issue. Can I get any solution for this?
1. Airflow Version : 2.0.1
2. platform: Azure AKS
3. Executor: KubernetesExecutor
4. Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.3", GitCommit:"c92036820499fedefec0f847e2054d824aea6cd1", GitTreeState:"clean", 
BuildDate:"2021-10-27T18:41:28Z", GoVersion:"go1.16.9", Compiler:"gc", Platform:"windows/amd64"}
5. Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.13", GitCommit:"aac5f64a5218b0b1d0138a57d273a12db99390c9", GitTreeState:"clean",
 BuildDate:"2021-01-18T07:43:30Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
 
 Also, attached scheduler log
[scheduler_logs.zip](https://github.com/apache/airflow/files/7538283/scheduler_logs.zip)
s.

**Comment by duc-luong-tnsl** [comment]
Hello saip50,
You said stuck it means your task is not changed state to `scheduled` or `queued `? 
Did you check log of scheduler?

**Reply by saip50** [reply]
Hi duc-luong-tnsl,
yes, when we are triggering tasks manually, it will remain in scheduled state itself, neither changed to queued nor running.
I have the logs of scheduler too. Please check it.
